packages/homepage/src/pages/embeds.js

```diff
@@ -18,10 +18,14 @@ import {
   SwitchWrapper,
   CustomLightStyles,
   Customizations,
+  Button,
 } from './_embeds.elements';
 
 export default () => {
   const [theme, setTheme] = useState('dark');
+  const [devTools, setDevTools] = useState(1);
```

Comment:
Maybe make this a boolean instead? 🤔

```ts
[hideDevTools, setHideDevtools] = useState(true);
```

That way setting the value will be just reverting the current value

```ts
<Button
  active={!hideDevTools}
  onClick={() => setHideDevtools(hide => !hide)}
  // ...
>
```